### PR TITLE
Update conferences.json

### PR DESCRIPTION
**Pull request by spastorino**
r? @XAMPPRocky

### CODE REVIEW

**Comment by XAMPPRocky** [comment]
cc @nellshamrell for the rustconf blurb

**Comment by spastorino** [comment]
Yeah, I guess RustConf organizers would want to add more "features" to the file and other things. Feel free to close this PR and open a new one.

**Comment by nellshamrell** [comment]
This is fine for now! I'll open a new PR if I want to add more in.

**Comment by XAMPPRocky** [comment]
Ok @spastorino Thank you for your PR, and congrats on your first contribution! 🎉

File: conferences.json [hunk 1]
```diff
@@ -210,5 +210,18 @@
             "hackaton": "yes",
             "live streaming": "yes"
         }
-    }
+    },
+    {
+        "name": "RustConf",
+        "date": "2020-08-20T09:30:00-07:00",
+        "blurb": "RustConf 2020 online",
+        "location": "Portland, USA",
```
Comment (spastorino) [inline conferences.json:218]: Just figured that it should've said online.

Comment (nellshamrell) [reply conferences.json:218]: Ah! Yes, that is correct, it is all online.

**Pull request merged**